### PR TITLE
Add 404 error page

### LINKED ISSUE #465: Add 404 Page
We need to add a better styled 404 page. 
Currently we have a very barebone page that says "Not Found" without any styling (example [here](http://elixir-companies.com/asdasdasdad))

### PR DESCRIPTION
**Pull request by Nitrino**
Resolve #465
localhost_4000_sss (2)

### CODE REVIEW

**Comment by doomspork** [comment]
@Nitrino: You. Are. Awesome! Seriously, you're awesome. 😁 

Thank you so much for doing this. I'm going to defer to @tajchumber and @burden, who're far more savvy at this stuff than myself, for the review.

File: assets/css/errors.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.not_found {
+  position: absolute;
+  top: 50%;
+  margin-left: 50%;
+  padding: 0.5rem 0.75rem;
+  transform: translateY(-50%) translateX(-50%);
+  text-align: center;
+  color: $purple;
+
+  &__code {
```
Comment (tajchumber) [inline assets/css/errors.scss]: throughout the project we don't use the `&__` notation, so to keep it consistent with every thing else, can you change it to `&-` please?

File: assets/css/errors.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    font-size: 10.8em;
+    font-weight: bold;
+    line-height: 1.25;
+  }
+
+  &__text {
```
Comment (tajchumber) [inline assets/css/errors.scss]: same here as above :)

**Comment by Nitrino** [comment]
@tajchumber fixed

**Comment by tajchumber** [comment]
thankk @Nitrino ! great job :)

**Comment by doomspork** [comment]
@Nitrino we have the exact same issue, even opened by @tajchumber himself, over at Elixir School if you're looking to continue your contributions https://github.com/elixirschool/elixirschool/issues/1730

Thanks again! 😁

**Comment by Nitrino** [comment]
@doomspork Of course, I will make PR to elixirschool

**Pull request merged**